monitors/BalanceMonitor.js

```diff
@@ -105,7 +107,7 @@ class BalanceMonitor {
       const monitoredAddress = this.web3.utils.toChecksumAddress(bot.address);
 
       if (this.toBN(this.client.getCollateralBalance(monitoredAddress)).lt(this.toBN(bot.collateralThreshold))) {
-        this.logger.warn({
+        this.logger[this.logOverrides.collateralThreshold ? this.logOverrides.collateralThreshold : "warn"]({
```

Comment:
I think a slightly more concise syntax for this would work: `this.logOverrides.collateralThreshold || "warn"`.

Reply:
Implemented this refactor for all bot logOverrides!